index.html

```diff
@@ -47,7 +72,27 @@
 
 	</div>
 
```

Comment:
I would suggest styling the ``contributors-container`` with the appropriate margin/padding, rather than using ``<br />`` tags.